system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -1683,9 +1690,12 @@ class CLdapClient : implements ILdapClient, public CInterface
             {
                 //Create HPCC admin group
                 {
-                    DBGLOG("Adding HPCC Admin group");
-                    try { addGroup(adminGroupName.str(), nullptr, "HPCC Administrators"); }
-                    catch(...) {}//group may already exist, so just move on
+                    if (!m_ldapconfig->getDisableCreateAdminGroup())
+                    {
+                        DBGLOG("Adding HPCC Admin group");
+                        try { addGroup(adminGroupName.str(), nullptr, "HPCC Administrators"); }
```

Comment:
Should this code check if the group already exists before adding (in the same way that createLdapBasedn does)  - otherwise each time an ldap security manager is created it will try and create the group.

Reply:
There is a comment in the catch block about ignoring if it already exists. This code was not changed. This process is used across the component when creating resources. No check is made for existence first, instead if it already exists, the error is handled and/or returned.